src/crud.js

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable */
+import { setStorage } from './storage.js';
+import { clearAll } from './clear.js';
+
+export { deleteTask, tasksArray, addTodo };
+
+const toDoList = document.getElementById('to-dos');
+const clearBtn = document.getElementById('clear-button');
+
+class Task {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+let tasksArray = [];
+const addTodo = (taskValue) => {
+  const localData = JSON.parse(localStorage.getItem('todo'));
+  localData.map((i) => tasksArray.push(i));
+  const taskContainer = document.createElement('div');
+  taskContainer.className = 'list-item';
+  taskContainer.innerHTML += `
+    <input type="checkbox" class="checkbox">
+    <span>${taskValue}</span>
+    <i class="fas fa-ellipsis-v"></i>
+    <i class="fas fa-trash-alt"></i>
+  `;
+  toDoList.appendChild(taskContainer);
+
+  const checkbox = document.querySelectorAll('.checkbox');
+  checkbox.forEach((i) => {
+    i.addEventListener('click', () => {
+      i.parentElement.classList.toggle('list-selected-item');
+      i.nextElementSibling.classList.toggle('text-selected');
+      i.parentElement.lastElementChild.classList.toggle('delete-icon');
+      i.parentElement.lastElementChild.previousElementSibling.classList.toggle('disable-edit');
+      updateStorage();
+    });
+  });
+
+  const newTask = new Task(taskValue, false, checkbox.length);
+  tasksArray.push(newTask);
+  setStorage(tasksArray);
+
+  const edit = document.querySelectorAll('.fa-ellipsis-v');
+  edit.forEach((i) => {
+    i.addEventListener('click', () => {
+      editTask(taskContainer, i.previousElementSibling);
+      i.parentElement.classList.add('list-selected-item');
+    });
+  });
+
+  const remove = document.querySelectorAll('.fa-trash-alt');
+  remove.forEach((i) => {
+    i.addEventListener('click', () => {
+      deleteTask(i.parentElement);
+    });
+  });
```

Comment:
When I tick a todo as completed and click on the delete icon, the todo is removed from the screen, and after that, when I refresh the browser the same todo that's supposed to have been deleted still shows up on the list again.

- [ ] Kindly work on the delete task so we can be able to delete a todo, it gets removed from the list on the frontend and as well from localStorage. So when we refresh the browser, the remaining todos are fetched and get populated on the screen or zero todos in a case where there was only one. 👍
- [ ] Make sure when we delete a todo, you update the localStorage list. 

Before delete

![image](https://user-images.githubusercontent.com/50941074/178038197-9f4e3b76-1fb0-4acf-adce-a9a79f52549c.png)

After delete, the same todo shows again

![image](https://user-images.githubusercontent.com/50941074/178038374-c8e68dec-0007-4e92-bbb3-36e95f675a89.png)